skimage/measure/__init__.py

```diff
@@ -1,3 +1,45 @@
+"""
+Image measurement tools. A collection of functions for image measurement.
+
+Functions
+---------
+find_contours : Find iso-valued contours in a 2D array for a given level value.
+regionprops : Calculate various properties of labeled image regions.
+regionprops_table: Return measurements for image region properties in tabular form.
+perimeter : Calculate perimeter of objects in the label image.
```

Comment:
I'm assuming that you copied the descriptions from their docstrings? If you look at the [rendered HTML for this PR](https://output.circle-artifacts.com/output/job/8a4465ec-8f86-4b6f-90f0-0c7a599184d6/artifacts/0/doc/build/html/api/skimage.measure.html) you see that we already have a list of all these functions generated automatically after this docstring. I don't think we really need to add this list here.

Reply:
Noted @lagru.